content/sdk/10.js/00.ethers/20.v6/03.providers.md

```diff
@@ -23,82 +23,42 @@ Use `BrowserProvider` for browser integrations.
 
 ### `constructor`
 
-Creates a zksync Era `Provider` object.
+Creates a ZKsync Era `Provider` object.
 
-**Parameters:**
+#### Inputs
 
-| Parameter  | Type                                                                                 | Description                                                |
-| ---------- | ------------------------------------------------------------------------------------ | ---------------------------------------------------------- |
-| `url?`     | [`ethers.FetchRequest`](https://docs.ethers.org/v6/api/utils/fetching/#FetchRequest) | Network RPC URL (optional).                                |
-| `network?` | [`ethers.Networkish`](https://docs.ethers.org/v6/api/providers/#Networkish)          | Network name, chain ID, or object with network details.     |
-| `options?` | `any`                                                                                | Additional provider options (optional).                    |
+| Parameter  | Type                                                                                   | Description                |
+| ---------- | -------------------------------------------------------------------------------------- | -------------------------- |
+| `url?`     | string or [`ConnectionInfo`](https://docs.ethers.org/v5/api/utils/web/#ConnectionInfo) | Network RPC URL (optional) |
+| `network?` | `ethers.providers.Networkish`                                                          | Network name (optional)    |
```

Comment:
Please double check that you are referring to the correct content for the section. This is V6 section but the content you changed look to be for V5.